packages/opentelemetry-api/src/trace/NoopTracer.ts

```diff
@@ -42,4 +63,13 @@ export class NoopTracer implements Tracer {
   }
 }
 
+function isSpanContext(spanContext: any): spanContext is SpanContext {
+  return (
+    typeof spanContext === 'object' &&
```

Comment:
should we rather fully validate here the spanId, traceId and traceFlags ?

why `spanContext['spanId']` instead of `spanContex.spanId` etc.

Reply:
It's valided above, this function is only used for the typeguard. Since the `parent` can be both a span or a spanContext